product-service/src/createProduct.js

```diff
@@ -0,0 +1,59 @@
+import response from "./utils/response";
+import isEmpty from "validator/es/lib/isEmpty";
+import isInt from "validator/es/lib/isInt";
+import { endClient, getClient, query, queryOne } from "./utils/db";
+import HttpError from "./utils/httpError";
+
+const validateData = (title, price, count) => {
+    const messages = {};
+    if (!title || isEmpty(title)) {
+        messages.title = 'Invalid title';
+    }
+
+    if (!price || !isInt(price, { min: 0 })) {
+        messages.price = 'Invalid price';
+    }
+
+    if (!count || !isInt(count, { min: 0 })) {
+        messages.count = 'Invalid count';
+    }
+
+    if (messages.length) {
+        throw new HttpError(422, messages.join('.'));
+    }
+}
+
+export const createProduct = async event => {
+    // noinspection JSUnresolvedVariable
+    console.log({
+        path: event.path,
+        method: event.httpMethod,
+        body: event.body
+    });
+    let client;
+    try {
+        const { title, description, price, count } = event.body;
```

Comment:
I think that it will be json string. Don't you need to parse it? `JSON.parse`

Reply:
Thank you, I will resolve it.

Reply:
Resolved